development/build/utils.js

```diff
@@ -261,32 +253,6 @@ function getBuildName({
   return name;
 }
 
-/**
- * Get the app ID for the current build. Should be valid reverse FQDN.
- *
- * @param {object} options - The build options.
- * @param {string} options.buildType - The build type of the current build.
- * @returns {string} The build app ID.
- */
-function getBuildAppId({ buildType }) {
-  const baseDomain = 'io.metamask';
-  return buildType === 'main' ? baseDomain : `${baseDomain}.${buildType}`;
-}
-
-/**
- * Get the image data uri for the svg icon for the current build.
- *
- * @param {object} options - The build options.
- * @param {string} options.buildType - The build type of the current build.
- * @returns {string} The image data uri for the icon.
- */
-function getBuildIcon({ buildType }) {
-  const svgLogoPath =
-    BUILD_TYPES_TO_SVG_LOGO_PATH[buildType] ||
-    BUILD_TYPES_TO_SVG_LOGO_PATH.main;
-  const svg = readFileSync(svgLogoPath, 'utf8');
-  return `data:image/svg+xml,${encodeURIComponent(svg)}`;
-}
```

Comment:
These were only used by `setEnvironmentVariables`, so I've moved them to the new `set-environment-variables.js` file.

Reason: `utils.js` is expensive to import, and the webpack build wouldn't have any use for anything else in `utils.js`.